packages/strapi-plugin-content-manager/services/data-mapper.js

```diff
@@ -98,6 +98,7 @@ const HIDDEN_CONTENT_TYPES = [
   'plugins::upload.file',
   'plugins::users-permissions.permission',
   'plugins::users-permissions.role',
+  'plugins::i18n.locale',
```

Comment:
We need to introduce something so this becomes dynamic we can't let that be static anymore

Reply:
What do you suggest?
An option in the models ? (I prefer this one)
A service where plugin can register the hidden models ?

Reply:
An option in the models is fine. Can you write up an ADR to document the existing options & propose a new or updated one so we can take a decision :)

Reply:
I'll do it in another PR :)